app/navigation/LocationOfGoodsNavigator.scala

```diff
@@ -36,6 +36,7 @@ class LocationOfGoodsNavigator @Inject() () extends Navigator {
     case IdentificationPage                                                                       => ua => routeIdentificationPageNavigation(ua, departureId, mode)
     case CountryPage                                                                              => ua => AddressPage.route(ua, departureId, mode)
     case MoreInformationPage                                                                      => ua => locationOfGoodsNavigation(ua, departureId, mode)
+    case CheckInformationPage                                                                     => ua => Some(controllers.routes.CheckYourAnswersController.onPageLoad(departureId))
```

Comment:
What is this page? It sounds like another CYA page

Reply:
It is the page that shows up when the answers from IE015/13 are checked and deemed to be complete. The MoreInformationPage is the one that shows up when the users have not provided all answers needed in the IE015/13 that comes through @joecefnhedog